knowledge/cars/delorean/qna.yaml

```diff
@@ -0,0 +1,152 @@
+  question: What kind of engine was the car intended to have originally?
+- answer: Legend Industries converted four Deloreans, two twin-turbo cars (VIN
+    502 and VIN 530) and two single-turbo cars (VIN 528 and VIN 558) into turbo
+    versions.
+  question: Were there any turbo DeLoreans produced?
+- answer: \"Black cars\" are pre-production units made with fiberglass bodies
+    in order to train the workforce.
+  question: What are Delorean "black cars"?
+- answer: The DeLorean featured gull-wing doors and was one of the first to do
+    so after the Mercedes 300SL.
+  question: What kind of doors did the DeLorean have?
+- answer: The DeLorean features heavy gull-wing doors supported by cryogenically
+    preset torsion bars and nitrogen-charged struts.
+  question: What is unique about the DMC DeLorean doors?
+- answer: Several dealer options were available, including a car cover, sheepskin
+    seat covers, floor mats, car care cleaning kit, black textured accent stripes,
+    grey scotch-cal accent stripes, a luggage rack and a ski-rack adapter.
+  question: What dealer options were available for the Delorean?
+- answer: DMC stands for DeLorean Motor Company.
+  question: What does DMC stand for?
+- answer: The Ford Cologne V6 engine.
+  question: Which engine was used in the DMC DeLorean?
+- answer: The Ford Cologne V6 engine was first considered. The 2.0 engine from
+    a the Citroen CX2000 was then considered and eventually dropped in favor
+    of the fuel injected v6 PRV (Peugeot-Renault-Volvo) engine.
+  question: What engines were considered in the making of the DMC Delorean?
+- answer: The DeLorean's body pannels are made of brushed SS304 austenitic
+    stainless steel.
+  question: What are the DeLorean's body panels made of?
+- answer: The DMC DeLorean was made popular by the movie Back to the Future
+  question: What made the DMC DeLorean so popular?
+- answer: The DMC DeLorean was made popular by the movie Back to the Future
+  question: How did the DMC DeLorean become so well known?
+- answer: The body is paneled in brushed SS304 austenitic stainless steel.
+  question: What materials are the DMC DeLorean body panels comprised of?
+- answer: The DeLorean has a four-wheel independent suspension with coil springs,
+    and telescopic shock absorbers. The front suspension uses double wishbones,
+    while the rear is a multi-link setup.
+  question: What kind of suspension did the DeLorean have?
+- answer: Stainless-steel panels were fixed to a fiberglass underbody. The
+    underbody was affixed to a steel backbone chassis with Y-frames at either
+    end, which was derived from the Lotus Esprit platform. The chassis was
+    coated with epoxy, a material to protect steel against corrosion.
+  question: What materials was the Delorean body made from?
+- answer: Upon release in 1981, a DeLorean had a base MSRP of $25,000, or equivalent
+    to $84,000 in 2023. MSRP increased in 1982 to $29,825, equivalent to $94,000
```

Comment:
The dollar figures here are from Wikipedia but are not all present in the Markdown

https://github.com/instructlab/taxonomy/pull/1156#issuecomment-2146928464